client/src/components/ui/css/LandingPage.css

```diff
@@ -0,0 +1,31 @@
+@import url("https://fonts.googleapis.com/css2?family=Michroma&display=swap");
+
+html,
+body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+/* Ensure full height for main container */
+main {
+  height: 100%;
+  display: flex;
+  align-items: left; /* Vertically center the content */
+  justify-content: left;
+}
+
+body {
+  background: linear-gradient(#00293f, 55%, #d3dee1);
```

Comment:
You can use this: 
https://chakra-ui.com/docs/styled-system/style-props#gradient